src/Microsoft.PackageManagement.CoreProviders/project.json

```diff
@@ -3,7 +3,8 @@
 	"version": "1.0.0-*",
 
 	"buildOptions": {
-		"warningsAsErrors": true
+		"warningsAsErrors": true,
+                "debugType": "portable"
```

Comment:
this `debugType` setting should go in `buildOptions` section within `netstandard1.6`, preferably right after the `define` setting.